Game/Board.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-import History, Pieces, Position
+
+from . import History, Pieces, Position
```

Comment:
I am unfamiliar with this from . Syntax. Is it functionally equivalent to just import? Perhaps for code clarity we make that change.